1-Initial-Setup/1.5-Additional-Process-Hardening.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+@test "1.5.1 Ensure XD/NX support is enabled (Manual)" {
+    run bash -c "journalctl | grep 'protection: active'"
+    [ "$status" -eq 0 ]
+}
+
+@test "1.5.2 Ensure address space layout randomization (ASLR) is enabled (Automated)" {
+    run bash -c "sysctl kernel.randomize_va_space"
+    [ "$status" -eq 0 ]
+    [[ "$output" == "kernel.randomize_va_space = 2" ]]
+    run bash -c "grep -Es \"^\s*kernel\.randomize_va_space\s*=\s*([0-1]|[3-9]|[1-9][0-9]+)\" /etc/sysctl.conf /etc/sysctl.d/*.conf /usr/lib/sysctl.d/*.conf /usr/local/lib/sysctl.d/*.conf /run/sysctl.d/*.conf"
+    [ "$status" -ne 0 ]
+    [[ "$output" == "" ]]
+}
+
+@test "1.5.3 Ensure prelink is disabled (Automated)" {
+    run bash -c "dpkg -s prelink | grep -E '(Status:|not installed)'"
+    [ "$status" -eq 1 ]
```

Comment:
```suggestion
    [ "$status" -eq 1 ]
    [[ "$output" == *"dpkg-query: package 'prelink' is not installed and no information is available"* ]]
```

Reply:
I think checking the exit code is already sufficient. The output check could go wrong on machines which are not configured to use the English language.

Reply:
The rest of our tests are also written in such a way that we test everything in english :grin:

Reply:
You're right :D